src/main/java/Hailey.java

```diff
@@ -0,0 +1,73 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Hailey {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        List<Task> tasks = new ArrayList<>();
```

Comment:
It is good to specify access modifiers for the variables being declared, like `private`,`public` or `protected`.